README.md

```diff
@@ -267,9 +271,28 @@ This assumes that Runic is installed in the `@runic` shared project as suggested
 (push '(julia-mode . runic) apheleia-mode-alist)
 ```
 
+### Git integration
+
+The [`git-runic`](https://github.com/fredrikekre/Runic.jl/blob/master/bin/git-runic)
+script (a modified version of
+[`git-clang-format`](https://github.com/llvm/llvm-project/blob/main/clang/tools/clang-format/git-clang-format))
+provides a convenient way to apply Runic formatting incrementally to a code
+base by limiting formatting to lines that are added or modified in each commit.
+The script can be installed with the following commands (replace the two
+occurences of `~/.local/bin` if needed):
+
+```sh
+# Download the script into ~/.local/bin
+curl -fsSL -o ~/.local/bin/git-runic https://raw.githubusercontent.com/fredrikekre/Runic.jl/refs/heads/master/bin/git-runic
+# Make the script executable
+chmod +x ~/.local/bin/git-runic
```

Comment:
Duplicated with the installation instructions at the top?

Reply:
That was on purpose. The instructions at the top are just a quick copy-paste thing but this here has some more details.